src/device/device.c

```diff
@@ -152,11 +145,25 @@ void init_device(struct device* dev,
         { A(MM_PI_REGS, 0xffff), M64P_MEM_PI, { &dev->pi, RW(pi_regs) } },
         { A(MM_RI_REGS, 0xffff), M64P_MEM_RI, { &dev->ri, RW(ri_regs) } },
         { A(MM_SI_REGS, 0xffff), M64P_MEM_SI, { &dev->si, RW(si_regs) } },
+        { A(MM_DOM2_ADDR1, 0xffffff), M64P_MEM_NOTHING, { NULL, RW(open_bus) } },
```

Comment:
Should we really be expanding usage of these macros? It feels like they really obfuscate the code.

Reply:
These macros allows for less repetition while still being somewhat readable.
Also they are scoped (undef'd after table declaration) to avoid unwanted usage.

Surely it is much easier to read/modify than what we had before the big memory refactoring :)

Reply:
Yes, true, easier to modify. But I almost have to manually expand the macro to wrap my head around. You do get used to it, but it's a pain at first.